Learn_go/网络通信/learn02-web/post/main.go

```
@@ -0,0 +1,23 @@
+
+// 发送post请求
+func main() {
+	postA()
+}
+
+func postA() {
+	path := "http://apis.juhe.cn/simpleWeather/quary"
+	values := url.Values{}
+	values.Set("city", "深圳")
+	r, _ := http.PostForm(path, values)
+	defer r.Body.Close()
```

golangci-lint (1.64.8)

httpresponse: using r before checking for errors

(govet)